addons/goggles/initSettings.sqf

```diff
@@ -0,0 +1,8 @@
+[
+    QGVAR(showClearGlasses),
+    "CHECKBOX",
+    [LSTRING(SettingShowClearGlasses), LSTRING(SettingShowClearGlasses)],
+    localize LSTRING(SettingsName),
+    false, // default value
+    2 // isGlobal
```

Comment:
Would `0` be better since it will still be client settable but can also still be forced by mission/server.

Reply:
Thanks for review, will apply suggestions soon. It would be cool if you would use GH suggestion feature in future, makes life easier 😜